### PR TITLE
Fix several issues

### PR DESCRIPTION
**Pull request by rhahne**
**IE11 polyfills**

I tested it on IE11 and had to include one more polyfill in order to make it work. I added it to the readme file if one wants to add it to the project.

**Export SCSS**

The scss file is now also exported. The corresponding SCSS variables are now overridable. I added a description on how to do that in the README file.

**CSS fix**

Also I fixed a minor css isse (with :focus-visible)

**version to v1.0.2**

As soon as this PR gots approved I will publish the new version v1.0.2 to the npm registry.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -19,8 +19,23 @@ Install with npm:
...
 The CSS is small enough to copy directly into your project's main stylesheet if you desire.
 
+CSS
+```bash
+import "~@yoo-digital/focus-overlay/dist/focusoverlay.css"
+```
+
+SCSS
+
+If you want to adjust the styling of the focus overlay import the SCSS file instead of the CSS file and override the [SCSS variables](https://github.com/yoo-digital/FocusOverlay/blob/master/src/styles.scss) as you wish.
```
Comment (kelvinlouis) [inline README.md]: Add a comma:
If you want to adjust the styling of the focus overlay,

File: README.md [hunk 2]
```diff
@@ -19,8 +19,23 @@ Install with npm:
...
+
+ ```bash
+import "~@yoo-digital/focus-overlay/src/styles.css"
```
Comment (kelvinlouis) [inline README.md]: incorrect file ending.

File: src/styles.scss [hunk 3]
```diff
@@ -33,8 +33,11 @@ $focus-overlay-transition: all 0.2s cubic-bezier(0, 1, 0, 1) !default;
 
   &-target {
     &,
-    &:focus,
-    &:focus:not(:focus-visible),
```
Comment (kelvinlouis) [inline src/styles.scss:37]: are you sure you do not want to state this separately for browsers that support :focus-visible?

**Review by kelvinlouis — CHANGES_REQUESTED** [review]

File: src/styles.scss (hunk 3 above)
Comment (rhahne) [reply src/styles.scss:37]: This is actually not necessary.

`&:focus:not(:focus-visible)` will be triggered for mouse users ONLY if
  :focus-visible is supported on this platform.

**Review by kelvinlouis — APPROVED** [review]

**Pull request merged**